flink-autoscaler-standalone/pom.xml

```diff
@@ -133,6 +133,12 @@ under the License.
             <version>${log4j.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
+            <version>${hikari.version}</version>
+        </dependency>
```

Comment:
If `flink-autoscaler` depends on `HikariCP`, and `flink-autoscaler-standalone` depends on `flink-autoscaler`.

Why does `flink-autoscaler-standalone` still need to depends on `HikariCP`? IIUC,  `flink-autoscaler-standalone`  already depends on `HikariCP` silently.

Please correct me if my understanding is wrong.

Reply:
I first passed the compilation, ran it locally myself and again saw if it worked as expected. I'm trying to remove that dependency to see if it's affected, and I'll come to a conclusion later

Reply:
Hi @czy006 , this PR is not updated more than 6 weeks, is there any conclusion about this comment and this PR?